src/index.html

```diff
@@ -13,10 +18,87 @@
     <title>Moyo header</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="style.css"
     />
   </head>
-  <body>
-    <h1>Moyo header</h1>
-  </body>
+  <header class="header">
+    <a class="logo">
+      <img
+        src="/moyo_1.png"
+        alt="Ikon"
+      />
+    </a>
+
+    <nav class="nav">
+      <ul class="nav-list">
+        <li class="nav-item">
+          <a
+            href="#"
+            class="nav-link is-active"
+          >
+            Apple
+          </a>
+        </li>
+        <li class="nav-item">
```

Comment:
```suggestion
        </li>
        
        <li class="nav-item">
```

<img width="832" alt="image" src="https://github.com/user-attachments/assets/e2b72e85-98f7-458a-8e3c-e72b6b7ffbe5">